modules/ding_base/ding_base.install

```diff
@@ -192,3 +194,10 @@ function ding_base_update_7006() {
   // Use the default version by setting an falsy value.
   _jquery_update_set_theme_version('seven', '');
 }
+
+/**
+ * Use jQuery 1.7 for admin theme.
+ */
+function ding_base_update_7007() {
+  _jquery_update_set_theme_version('seven', '1.7');
```

Comment:
_jquery_update_set_theme_version() is defined in jquery_update.install
it won't be available for ding_base.install, will it?